templates/includes/card-group-record-summary-no-image.html

```diff
@@ -8,19 +8,19 @@
 
 <li class="col-sm-12 col-md-6 col-lg-4">
     <div class="card-group-record-summary">
-        <a href="{% record_url record is_editorial=True %}" class="card-group-record-summary__link" data-card-type="card-group-record-summary-no-image" data-card-title="{{ record.title }}"{% if FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %} target="_blank"{% endif %}>
+        <a href="{% record_url record is_editorial=True %}" class="card-group-record-summary__link" data-card-type="card-group-record-summary-no-image" data-card-title="{% firstof description_override|safe record.title|striptags|linebreaks %}"{% if FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %} target="_blank"{% endif %}>
             <p class="card-group-record-summary__heading">
                 {% if record.reference_number %}
                 <span class="sr-only">
                     {{ record.reference_number }} -
                 </span>
                 {% endif %}
-                {{ record.title }}
+                {% firstof description_override|safe record.title|striptags %}
                 {% if FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %}<span class="sr-only">(link opens in a new window)</span>{% endif %}
             </p>
         </a>
         <div class="card-group-record-summary__body">
-            <p>{{ description_override|default:record.description|striptags|truncatewords:40 }}</p>
+            <p>{{ record.description | striptags | escape }}</p>
```

Comment:
I'm unsure of the need for the `| escape` addition here. If we aren't marking the value as 'safe', it should be escaped automatically here.
```suggestion
            <p>{{ record.description|striptags }}</p>
```